.github/workflows/ci.yml

```diff
@@ -7,6 +7,7 @@ on: # yamllint disable-line rule:truthy rule:comments
   push:
     branches:
       - "master"
+      - "jorlandobr-patch-2"
```

Comment:
```suggestion

```

Reply:
Please also remove the patch-3 from the PR.

Reply:
Removed!

Reply:
> 
Closed by accident! Could yoou please reopen it and merge?